### ISSUE TITLE
Duty list background causing access violation

### ISSUE THREAD

**Issue by Scrxtchy** [Caraxi/SimpleTweaksPlugin#571]
The feature introduced in #566 causes a access violation when transporting between zones.
if I turns the tweak on, or even if I turn it off again after the fact, the error is still presented

<details>
<summary>crash log</summary>

```
Unhandled native exception occurred at ffxiv_dx11.exe+4F6A00	(sub_1404F69E0+0x20)
Code: C0000005
Dump at: C:\Users\...\AppData\Roaming\XIVLauncher\addon\Hooks\7.5.1.0\..\..\..\dalamud_appcrash_20230510_171109_034_61112.dmp
Time: 2023-05-10 17:11:10.1707725 AEST

   at System.Environment.get_StackTrace()
   at Dalamud.EntryPoint.VehCallback() in C:\goatsoft\companysecrets\dalamud\EntryPoint.cs:line 71
   at Dalamud.Game.Framework.HandleFrameworkUpdate(IntPtr framework) in C:\goatsoft\companysecrets\dalamud\Game\Framework.cs:line 487

Exception Info #0
Address: C0000005
Flags: 0
Address: 7FF6735E6A00
Parameters: 0, 7FF600000034
Call Stack
{
  [0]	ffxiv_dx11.exe+4F6A00	(sub_1404F69E0+0x20)
  [1]	ffxiv_dx11.exe+4F6A00	(sub_1404F69E0+0x20)
  [2]	ffxiv_dx11.exe+4F7355	(sub_1404F72F0+0x65)
  [3]	ffxiv_dx11.exe+5870C5	(sub_140586FC0+0x105)
  [4]	ffxiv_dx11.exe+553CE4	(sub_140553B80+0x164)
  [5]	ffxiv_dx11.exe+5535E8	(sub_140553510+0xD8)
  [6]	ffxiv_dx11.exe+511B89	(sub_140511B60+0x29)
  [7]	ffxiv_dx11.exe+1151C4A	(sub_141151AF0+0x15A)
  [8]	ffxiv_dx11.exe+114FF0B	(Client::UI::AddonToDoList.OnRequestedUpdate+0x2FB)
  [9]	ffxiv_dx11.exe+52BABA	(Component::GUI::AtkUnitManager.UpdateAddonByID+0x9A)
  [10]	ffxiv_dx11.exe+524A42	(sub_140524A10+0x32)
  [11]	ffxiv_dx11.exe+5256BF	(sub_1405255B0+0x10F)
  [12]	ffxiv_dx11.exe+52B11A	(Component::GUI::AtkUnitManager.vf5+0x2EA)
  [13]	ffxiv_dx11.exe+5519B4	(sub_1405518D0+0xE4)
  [14]	ffxiv_dx11.exe+60F07C	(Client::UI::UIModule.Update+0x1FC)
  [15]	ffxiv_dx11.exe+94A6F	(sub_140094A10+0x5F)
  [16]	ffxiv_dx11.exe+18D2FD	(Client::System::Framework::TaskManager::RootTask.vf1+0x2D)
  [17]	ffxiv_dx11.exe+18D533	(sub_14018D500+0x33)
  [18]	ffxiv_dx11.exe+92E0F	(Client::System::Framework::Framework.Tick+0x23F)
  [19]	7FFBF480CCB4
  [20]	A643C6048AB
  [21]	88DAC400
  [22]	7B628FF4F0
  [23]	1A4CA695658
  [24]	F2B6E8039892
  [25]	coreclr.dll+40ABB8	(coreclr_shutdown+0x4B478)
  [26]	FFFFFFFFFFFFFFFF
}

Registers
{
  RAX:	1E5207DAA10 [1E5784EE7A0]
  RBX:	0
  RCX:	1E5F5F16398 [7FF60000000C]
  RDX:	0
  R8:	7FF60000000C
  R9:	1
  R10:	1E5B9F17600 [14000B]
  R11:	1
  R12:	2
  R13:	0
  R14:	66
  R15:	64
  RSI:	1E5F5F16398 [7FF60000000C]
  RDI:	1E551D82AA8 [61639CB]
  RBP:	0
  RSP:	7B628FE6E0 [1E5B9672184]
  RIP:	ffxiv_dx11.exe+4F6A00	(sub_1404F69E0+0x20)
}

Stack
{
  [RSP+0]	1E5B9672184
  [RSP+8]	1E5B9671EE0 [ffxiv_dx11.exe+1A58BF0	(vtbl_Client::UI::AddonToDoList)]
  [RSP+10]	1E5AAB4D810 [ffxiv_dx11.exe+1933A40	(vtbl_Component::GUI::AtkComponentBase)]
  [RSP+18]	0
  [RSP+20]	1E5F5F16390 [ffxiv_dx11.exe+1932D10	(vtbl_Component::GUI::AtkTexture+0x18)]
  [RSP+28]	ffxiv_dx11.exe+4F7355	(sub_1404F72F0+0x65)
  [RSP+30]	1E551D82AA8 [61639CB]
  [RSP+38]	1E5F5F16390 [ffxiv_dx11.exe+1932D10	(vtbl_Component::GUI::AtkTexture+0x18)]
  [RSP+40]	1E5AAB49AD0 [ffxiv_dx11.exe+1933768	(vtbl_Component::GUI::AtkCollisionNode)]
  [RSP+48]	ffxiv_dx11.exe+11532A0	(sub_141152D70+0x530)
  [RSP+50]	B
  [RSP+58]	ffxiv_dx11.exe+5870C5	(sub_140586FC0+0x105)
  [RSP+60]	1E57322E7F0 [1E5B9736D28]
  [RSP+68]	2
  [RSP+70]	1E5B9672184
  [RSP+78]	7FF6FFFFFF00
}
```
</details>

**Comment by MidoriKami** [comment]
The only thing I can think of is depth-based priority causing problems.

Without depth-based priority the duty list icons render behind the texture while the text renders above.

**Comment by MidoriKami** [comment]
Potentially Addressed in #572
